### ISSUE TITLE
Question: cell comments and images

### ISSUE THREAD

**Issue by unrealsolver** [pyexcel/pyexcel#286]
Can I read cell comments and cell image from the .ods file? I didn't find anything in the code nor in the docs.

**Comment by chfw** [comment]
I don’t think so! This library meant to process easily available data in spreadsheets.
